src/compas_ghpython/helpers/__init__.py

```diff
@@ -5,6 +5,13 @@
 
 .. currentmodule:: compas_ghpython.helpers
 
+.. autosummary::
+    :toctree: generated/
+    mesh_draw
```

Comment:
not sure but you might have to add a blank line here

Reply:
That worked! Weird that compas_ghpython.geometry worked without the line break, I added it to both just in case.